ibis/backends/pandas/execution/generic.py

```diff
@@ -761,7 +764,7 @@ def execute_null_if_zero_series(op, data, **kwargs):
 
 @execute_node.register(ops.StringSplit, pd.Series, (pd.Series, str))
 def execute_string_split(op, data, delimiter, **kwargs):
-    return data.str.split(delimiter)
+    return pd.Series(map(lambda s: np.array(s.split(delimiter)), data))
```

Comment:
```
 s.apply(lambda x: np.array(x.split(delimiter)))
```
I think you can do this

Reply:
This was a bit tricky for performance reasons—using `Series.apply` seems to involve some Pandas-related overhead for each iteration. On the other hand, doing the iteration without using `Series.apply` and then wrapping it back into a Pandas Series was faster.

[Here are some quick benchmarks I had done](https://gist.github.com/timothydijamco/fc024b01df908b88e4090f31f4d75302) (~2s for the `map` approach vs ~10s for `Series.apply`)

Reply:
Please put a comment describing this perf issue.